Installing-and-Configuring/Flashing-Guides/x230.md

```diff
@@ -28,10 +35,28 @@ fairly easy to remove.
 Pull up the black plastic on the bottom right of the palm rest to reveal the two
  SPI flash chips.
 
+![Flash chips]({{ site.baseurl }}/images/Flash_chips.jpg)
+
 The top chip is 4MB and contains the BIOS and reset vector. The bottom chip is
  8MB and has the [Intel Management Engine (ME)](https://www.flashrom.org/ME)
   firmware, plus the flash descriptor.
 
+Based on [the work done here](https://github.com/osresearch/heads/issues/716),
+ the chips should be one 4M and one 8M of the following:
+
+<!-- markdownlint-disable MD013 -->
+|Vendor|Device| size|
+|--|--|--|
+|Eon | EN25QH32 | 4M|
+|Eon| EN25QH64 | 8M|
+|Macronix|MX25L3206E - MX25L3206E/MX25L3208E|4M|
+|Macronix|MX25L6405,MX25L6405D, MX25L6406E/MX25L6408E, MX25L6436E/MX25L6445E/MX25L6465E/MX25L6473E/MX25L6473F, MX25L6495F|8M|
+|Micron/Numonyx/ST|N25Q032..1E,  N25Q032..3E|4M|
+|Micron/Numonyx/ST N25Q064..1E,  N25Q064..3E|8M|
+|Winbond | W25Q32.V, W25Q32.W | 4M|
+|Winbond | W25Q64.V, W25Q32.W  | 8M|
+<!-- markdownlint-enable MD013 -->
+
 Try to read the name on the top SPI flash chip. Then, connect the clip and
```

Comment:
May be not clear enough for new users. Calling flashrom -p ch341_a with the clip connected should report a list of discovered SPI chips, where per board configuration should maybe contain the list of possible SPI chips?

Reply:
@tlaurion  I agree but I am not sure how to write clear enough flashing documentation.  I was hoping when additional flashing guides are written, they could be normalized.